src/message.rs

```diff
@@ -79,6 +79,7 @@ impl Message {
                 encoder.str("channelId").unwrap();
                 encoder.str(document_id.0.as_str()).unwrap();
                 encoder.str("message").unwrap();
+                encoder.tag(minicbor::data::Tag::Base64).unwrap();
```

Comment:
I think that in order to produce the same messages as the JS version this should probably be `Tag::Unassigned(64)`

Reply:
Done.